project/Http4sPlugin.scala

```diff
@@ -356,6 +356,7 @@ object Http4sPlugin extends AutoPlugin {
     val reactiveStreams = "1.0.3"
     val quasiquotes = "2.1.0"
     val scalacheck = "1.15.1"
+    val scalacheckEffect = "0.6.0"
```

Comment:
Oh, is this the missing piece on #3965?

Reply:
Not compeletly. This let's you evaluate properties running on `F` but we also need (I think) a`discipline-cats-effect` that let's you run sets of properties a.k.a `laws`. I started working on this but needs some extra work

Reply:
We could go the route of one IO result per test